sdks/python/apache_beam/runners/interactive/cache_manager_test.py

```diff
@@ -215,6 +215,21 @@ def test_load_saved_pcoder(self):
         type(self.cache_manager.load_pcoder('full', 'a key')),
         type(coders.registry.get_coder(int)))
 
+  def test_cache_manager_uses_ib_specified_cache_dir(self):
```

Comment:
Could you please add an additional test for both cache manager implementations when the cache dir is explicitly set to a local file path?

For example, "/tmp/it-test/", then you can actually verify if a cache dir and additional cache files are created by creating a pipeline with interactive runner and call ib.collect.

Reply:
I've added two additional tests:
- test under cache_manager_test.py that: 
  - writes some values to a cache with default settings
  - writes the same values to a cache with ib.options.cache_root
  - asserts that both caches contain the same contents
- test under streaming_cache_test.py that:
  - writes some values to a cache with default settings
  - writes the same values to a cache with ib.options.cache_root
  - asserts that both caches contain the same contents